### PR TITLE
segragation of styles to render in frontend

### PR DESCRIPTION
**Pull request by Yajme**
frontend login validation
added module for mysql

### CODE REVIEW

File: db/connection.mjs
Comment (Yajme) [inline db/connection.mjs:1]: This is for mysql connection

File: index.js
Comment (Yajmee) [inline index.js:1]: added static path for resource rendering

**Review by Yajmee — CHANGES_REQUESTED** [review]
the file in dashboard is missing. 

```
./views/student/index.ejs
and 
./views/student/dashboard.ejs

```
is technically the same

**Review by DzeyEm — APPROVED** [review]
Approved

File: index.js
Comment (Yajme) [reply index.js:1]: This will be fixed next commit/pull request

**Review by Yajmee — APPROVED** [review]

**Pull request merged**